host/sgx/calls.c

```diff
@@ -339,6 +339,7 @@ static const char* oe_ocall_str(oe_func_t ocall)
         "FREE",
         "SLEEP",
         "GET_TIME",
+        "FUTEX_WAKE",
```

Comment:
Isn't "FUTEX_WAKE" Linux-specific?   I'd prefer we not have Linux-specific names in non-Linux-specific files.

Reply:
On Windows I suppose this will use **WaitOnAddress**. Maybe "WAIT_ON_ADDRESS" would be more descriptive?

Reply:
Fixed. Changed to `WAKE_HOST_WORKER` to match enum name.